mix.exs

```diff
@@ -61,20 +61,30 @@ defmodule MMDB2Decoder.MixProject do
 
   defp docs do
     [
-      main: "MMDB2Decoder",
+      extras: [
+        "CHANGELOG.md",
+        "LICENSE": [title: "License"],
+        "README.md": [title: "Overview"]
+      ],
+      main: "readme",
       source_ref: "master",
-      source_url: @url_github
+      source_url: @url_github,
+      formatters: ["html"]
     ]
   end
 
   defp elixirc_paths(:test), do: ["lib", "test/helpers"]
   defp elixirc_paths(_), do: ["lib"]
 
   defp package do
-    %{
+    [
+      description: "MMDB2 File Format Decoder",
       files: ["CHANGELOG.md", "LICENSE", "mix.exs", "README.md", "lib"],
-      licenses: ["Apache 2.0"],
-      links: %{"GitHub" => @url_github}
-    }
+      licenses: ["Apache-2.0"],
+      links: %{
+        "Changelog" => "https://hexdocs.pm/mmdb2_decoder/changelog.html",
```

Comment:
Would you mind moving this to an `@url_changelog` attribute next to the `@url_github` just to keep the URLs together?